src/main/java/core/basesyntax/servise/implementations/FileServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.servise.implementations;
+
+import core.basesyntax.servise.FileService;
+import core.basesyntax.validators.DataValidator;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileServiceImpl implements FileService {
+    private static final String TO_FILE_PATH = "src/main/resources/Report";
```

Comment:
it's not `.csv` file
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/ee1b6b5a-d006-484d-a2e7-e67cf047c5ce)